test/test_tree_3d_mhd.jl

```diff
@@ -184,9 +184,9 @@ end
     end
 end
 
-@trixi_testset "elixir_mhd_alfven_wave.jl with Orszag-Tang setup + flux_hll" begin
+@trixi_testset "elixir_mhd_alfven_wave.jl with Orszag-Tang setup + FluxHLL(min_max_speed_naive)" begin
```

Comment:
I'm not sure whether we really want to test the naive version here or the Davis version.